app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -577,6 +577,14 @@ public static void reportNavDrawerItemSelected(String selectedItem) {
                 new String[]{selectedItem});
     }
 
+    public static void reportOtpRequested(int numberOfReattempts) {
```

Comment:
we should ensure this gets called everytime we request otp and probably better to rephrase `OTP_REATTEMPTS` to `otp_attempts` and make it more transparent to analytics users by simply tracking nth user attempt rather (n-1)th.